### PR TITLE
Emit event in systems

### PR DESCRIPTION
**Pull request by Maxteke**

### CODE REVIEW

File: libecs/CMakeLists.txt [hunk 1]
```diff
@@ -35,65 +35,65 @@ fetch_boost_library(predef)
...
+        source/Core/Data/type.cxx
+        source/Core/Event/index.cxx
+        source/Core/Event/description.cxx
+        source/Core/Event/manager.cxx
+        source/Components/TagArray.cxx
+        source/Components/Tag.cxx
+        source/Components/RigidBody.cxx
+        source/Components/Gravity.cxx
 )
 target_include_directories(${PROJECT_NAME} PUBLIC include/)
```
Comment (zcorniere) [inline libecs/CMakeLists.txt:73]: Can you remove the formatting change, please? Use cmake-format

Comment (Maxteke) [reply libecs/CMakeLists.txt:73]: Sure

**Review by zcorniere — APPROVED** [review]

File: libecs/CMakeLists.txt [hunk 2]
```diff
@@ -94,6 +94,6 @@ build_tests(
     tests/Core/Systems/test_description.cxx
     tests/Core/Event/test_description.cxx
     tests/Core/Event/test_manager.cxx
+    tests/Core/Event/test_child_event.cxx
     tests/Core/Scene/test_load.cxx
-    tests/Core/Scene/test_save.cxx
```
Comment (Ersikan) [inline libecs/CMakeLists.txt:98]: Why did you remove this test ?

Comment (Maxteke) [reply libecs/CMakeLists.txt:98]: Mistake :|

**Review by zcorniere — APPROVED** [review]
